tycho-client/Cargo.toml

```diff
@@ -33,6 +33,8 @@ tracing-subscriber = { version = "0.3.17", default-features = false, features =
 ] }
 clap = { workspace = true, features = ["derive", "env"] }
 tracing-appender = { workspace = true }
+# link openssl statically
+openssl = { version = "0.10", features = ["vendored"] }
```

Comment:
Maybe feature gate this?

Reply:
I found a way to conditionally include it depending on the target os. Thought that was cleaner than feature gating it - let me know what you think.